aptos-move/framework/aptos-stdlib/sources/math64.move

```diff
@@ -37,6 +37,15 @@ module aptos_std::math64 {
         large
     }
 
+    /// Returns least common multiple of `a` & `b`.
+    public inline fun lcm(a: u64, b: u64): u64 {
+        if (a == 0 || b == 0) {
+            0
```

Comment:
Should it return 0 or abort when a == b == 0?

Reply:
according to [wikipedia](https://en.wikipedia.org/wiki/Least_common_multiple#:~:text=Using%20the%20greatest%20common%20divisor,-The%20least%20common&text=However%2C%20if%20both%20a%20and,considered%20as%20a%20special%20case.), lcm(0, 0) is a special case, and should be considered as 0.

some popular languages such as [python](https://github.com/python/cpython/blob/0dcbc8385322ff51f7fc3e586027d880275df4fa/Modules/mathmodule.c#L815) also accepts `lcm(0, 0) = 0`

Reply:
hi @junkil-park, is there anything else?

Reply:
@movekevin @wrwg @davidiw @junkil-park

Reply:
The CI tests are running.

Reply:
I guess some tests are awaiting approval

Reply:
@junkil-park hi, any update?

Reply:
The CI tests passed. Two approvals are required to merge PRs. I've already approved it, and it needs one more approval.

Reply:
@davidiw @movekevin @wrwg UP

Reply:
@junkil-park  @davidiw @movekevin @wrwg

I was planning to contribute Aptos more, but seems like its not even open source 🤣🤣🤣